dataeng/jobs/analytics/ExpireVerticaPassword.groovy

```diff
@@ -0,0 +1,71 @@
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+
+
+class ExpireVerticaPassword {
+
+    public static def job = { dslFactory, allVars ->
+
+        dslFactory.job('expire-vertica-password') {
+            logRotator common_log_rotator(allVars)
+            parameters {
+                stringParam('TOOLS_REPO', allVars.get('TOOLS_REPO_URL'), '')
+                stringParam('TOOLS_BRANCH', allVars.get('TOOLS_BRANCH', 'origin/master'), 'e.g. tagname or origin/branchname')
+                stringParam('CREDENTIALS', allVars.get('CREDENTIALS'))
+                stringParam('EXCLUDE', allVars.get('EXCLUDE'))
+                stringParam('MAPPING', allVars.get('MAPPING'))
+                stringParam('NOTIFY', '$PAGER_NOTIFY', 'Space separated list of emails to send notifications to.')
+            }
+            multiscm {
+                git {
+                    remote {
+                        url('$TOOLS_REPO')
+                        branch('$TOOLS_BRANCH')
+                        credentials('1')
+                    }
+                    extensions {
+                        relativeTargetDirectory('analytics-tools')
+                        pruneBranches()
+                        cleanAfterCheckout()
+                    }
+                }
+            }
+            //triggers common_triggers(allVars)
+            wrappers {
+                timestamps()
+            }
+            publishers common_publishers(allVars)
+            publishers {
+                extendedEmail {
+                    recipientList('$NOTIFY_LIST')
+                    defaultSubject('You Vertica password has expired')
+                    defaultContent('This is to inform you that you Vertica password has expired. ' +
+                    'Please use vsql to set a new password.')
+                    triggers {
+                        success {
+                            sendTo {
+                                recipientList()
```

Comment:
Is there a way to make this send via Bcc?

Reply:
Yes, by prepending bcc: to the email, that is, bcc:someone@example.com. I'll make the change in the script.